### PR TITLE
Move Gymnasium production to new CMS service

### PR DESCRIPTION
**Pull request by mbifulco**
attn: @tkeemon 

# What this PR does
- Moves gymnasium's CMS source from the old github pages URLs to gymcms.xyz - with some fancy new mako syntax 

# Dependencies
- This change requires an `APPSEMBLER_FEATURES.ENVIRONMENT` variable to be set via `lms.env.json` or ansible, just like on staging - the value for production should be `production`.

# To test that this was a successful update
after compiling the theme and running the platform:
- the site shouldn't look any different visibly, but if you view the page source for compiled pages in the browser, you'll see comments like this:
image
Which indicate content is being pulled from our new CMS.  If they're showing up, everything is :100:.  If not, please roll this PR back for the time being, and we can investigate what's gone wrong.
- See files changed in this PR for which templates use these CMS imports - there are a few scattered over most pages of the site, in various capacities.

# Timeline 
- Once the ansible changes are in place, this should be good to go

### CODE REVIEW

**Comment by tkeemon** [comment]
[Aquent: review PR #79](https://trello.com/c/273fgEZv/2701-aquent-review-pr-79)

File: lms/templates/static_templates/careers.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
       <div class="white-panel">
         <div class="row">
           <div class="col-md-9">
-            ${urllib2.urlopen('https://gymnasium.github.io/static-site-content/careers.html').read()}
+            ${gymcms.render('careers')}
```
Comment (tkeemon) [inline lms/templates/static_templates/careers.html]: This template gives a 404 for both staging and prod: 

- https://staging.gymcms.xyz/static/careers
- https://gymcms.xyz/static/careers

File: lms/templates/static_templates/honor.html [hunk 2]
```diff
@@ -10,7 +10,7 @@
         <div class="row">
           <div class="col-md-9">
             ## include privacy policy from git repo
-            ${urllib2.urlopen('https://gymnasium.github.io/static-site-content/privacy-policy.html').read()}
+            ${gymcms.render('faq')}
```
Comment (tkeemon) [inline lms/templates/static_templates/honor.html]: Did you mean to render the FAQ page here? Would the Privacy Policy page be better?

File: lms/templates/login.html [hunk 3]
```diff
@@ -1,12 +1,13 @@
 <%inherit file="main.html" />
```
Comment (tkeemon) [inline lms/templates/login.html]: This file isn't going to be rendered because your site uses the combined Logistration page (the Backbone.js page) instead of this old HTML template. That being said, these changes aren't causing any problems, so you can leave it if you'd like. Or you can just nuke this file all together.

File: lms/templates/register.html [hunk 4]
```diff
@@ -1,6 +1,7 @@
 <%inherit file="main.html" />
 <%namespace name='static' file='static_content.html'/>
 <%namespace file='main.html' import="login_query"/>
+<%namespace file="./util/gymcms.mako" />
```
Comment (tkeemon) [inline lms/templates/register.html]: You're missing the `name='gymcms'` part of the import. 

But, the same comment from login.html applies here. This template is never being rendered because you're using the Logistration page. Keep the changes, revert the changes, or nuke the file all together. It doesn't make much of a difference.

**Review by tkeemon — CHANGES_REQUESTED** [review]
@mbifulco the changes look good. I have concerns about the careers.html and honor.html pages. After we have those sorted out, I'll merge this and push to prod.

File: lms/templates/static_templates/careers.html (hunk 1 above)
Comment (mbifulco) [reply lms/templates/static_templates/careers.html]: Good catch - updating this.

File: lms/templates/static_templates/honor.html (hunk 2 above)
Comment (mbifulco) [reply lms/templates/static_templates/honor.html]: This should have been the privacy policy - likely a dumb c/p job.  We don't actually link to an honor page anywhere on the site, handily, but it's good to do it to be thorough. I'll submit this change too.

File: lms/templates/login.html (hunk 3 above)
Comment (mbifulco) [reply lms/templates/login.html]: I like the idea of removing this template.  Will do.

File: lms/templates/register.html (hunk 4 above)
Comment (mbifulco) [reply lms/templates/register.html]: 👋 bye template!

**Comment by mbifulco** [comment]
@tkeemon thanks for your feedback here - always good to have a second set of 👀 on me.  I believe I've addressed everything you mentioned.  This should be good to go 👍

**Review by tkeemon — APPROVED** [review]
:+1: 

Changes look great. I'll merge this now, then push to prod late tonight (ET).
